README.md

```diff
@@ -1,8 +1,8 @@
 # PLCrashReporter
 
-Reliable, open-source crash reporting for iOS, Mac OS X and tvOS.
```

Comment:
Should we keep short summary?

Reply:
`summary` is a required property.

Reply:
Do you really want to keep this? This summary is just meaningless and description repeats what it says there.